### ISSUE TITLE
Watch app not working

### ISSUE THREAD

**Issue by maxito71** [itsMeDavidV/Watchla-Support#414]
Hi. I set the app on my iPhone but the app on the watch doesn’t work.
It says I just have to open the app on the phone but nothing happens.
Already tried to uninstall it and do everything from the beginning.
Nothing seems to work.
Thx

**Comment by CassandraMarie** [comment]
Hi @maxito71, we're happy to assist. Could you provide more details about the issue you're encountering? If possible, please include any error codes or messages you're seeing.

**Comment by maxito71** [comment]
Hi.
Thanks for the quick reply.
Here attached what my watch says and a screenshot of the phone app which,
as you can see, is connected.

Can’t do anything more than that.
Waiting

— max



Il giorno gio 3 ott 2024 alle 19:12 Cassandra Marie <
***@***.***> ha scritto:

**Comment by CassandraMarie** [comment]
Hi @maxito71, it appears that your attachment did not come through. Could you please try sending it again?

**Comment by maxito71** [comment]
Now?


— max


Il giorno gio 3 ott 2024 alle 19:44 Cassandra Marie <
***@***.***> ha scritto:

**Comment by CassandraMarie** [comment]
No, I don't see any attachments. Does the message have an error code or does it say authenticate your watch?

**Comment by maxito71** [comment]
“To sign in please authenticate in the Watchla iOS app”

— max


Il giorno gio 3 ott 2024 alle 20:37 Cassandra Marie <
***@***.***> ha scritto:

**Comment by CassandraMarie** [comment]
Could you please verify that you are using the latest version of Watchla, which is 3.8.4? To check, open the Watchla app on your Apple Watch, go to the 'Preferences' screen, and scroll down past the 'Sign Out' button until you see the 'app version' label.

The app version will be displayed in the following format: version (build number).

**Comment by maxito71** [comment]
On the watch I can do absolutely NOTHING: when I open it I got the screen
with the aforementioned request. By the way I guess it’s latest ‘cause I
uninstalled it from the watch and installed it again.
The version should depend on the Phone app… right?

— max


Il giorno gio 3 ott 2024 alle 21:02 Cassandra Marie <
***@***.***> ha scritto:

**Comment by CassandraMarie** [comment]
I apologize, I meant on your iPhone. Open the Watchla app on your iPhone, go to the 'Preferences' screen, and scroll down past the 'Sign Out' button until you see the 'App version' label.

**Comment by maxito71** [comment]
3.8.4 (494)

— max


Il giorno gio 3 ott 2024 alle 21:08 Cassandra Marie <
***@***.***> ha scritto:

**Comment by CassandraMarie** [comment]
I have a few troubleshooting options for you. 
1. Refresh your watch app - Open Preferences > Refresh watch app. Make sure to keep Watchla open on your Apple Watch while doing this. 
2. If refreshing the app doesn't work, force close out of Watchla on your watch and reopen it. 
3. If steps 1 & 2 don't work, sign out of Watchla on your iPhone and sign in again. 

Let me know if this helps!

**Comment by maxito71** [comment]
I’ve done the 3 procedures over and over, before contacting you.
Anything else?

— max


Il giorno gio 3 ott 2024 alle 21:24 Cassandra Marie <
***@***.***> ha scritto:

**Comment by CassandraMarie** [comment]
We apologize for the inconvenience. Watchla is currently experiencing an intermittent outage, which is causing the authentication issues. Our team is actively working to resolve the problem as quickly as possible.

I will keep you updated.

**Comment by CassandraMarie** [comment]
Can you send over your support code via [https://watchla.app/contact ](https://watchla.app/contact ) so we can investigate your case?

Support codes can be found by going to Preferences > Support/ Feedback > Support code
Please be sure to link to this issue when you reach out.

**Comment by maxito71** [comment]
Done.
Wrote this message:





*As requested by the support Cassandra tried to give me after I contacted
Watchla support via Github, here is my SUPPORT CODE: [redacted]  a quick
recap: can't make the app on my Apple Watch working. Stuck on the screen
"To sign in... Please Authenticate in the Watchla iOS app".The app on the
iPhone is working (I can see vehicle data) but could't set the virtual
key.Tried several times to "refresh watch app", uninstall and re-install,
logout and login... everything.Hope you could help*

Waiting



— max


Il giorno gio 3 ott 2024 alle 23:07 Cassandra Marie <
***@***.***> ha scritto:

**Comment by maxito71** [comment]
UPDATE
I had to restore to factory settings my watch and now it seems to work.
Best Regards
-----------------
M. Schenetti


Il giorno ven 4 ott 2024 alle ore 09:24 Max Schenetti ***@***.***>
ha scritto:
